### DISCUSSION TITLE
Mercury Theme: If Image zoom effect is set to 1, it still has effect on the heading of the post

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1449] [Bug Report] [answered]
### Operating system

macOS Sonoma

### Publii version

045.2

### Issue type

Free theme

### Bug description

Im Mercury 2.7.0.0 I set the "Image zoom effect" to 1, but the the heading of the post still moves up and down on mouse over. The same issue was already in version 2.6.0.0.

![Bildschirmfoto 2024-03-22 um 09 30 06](https://github.com/GetPublii/Publii/assets/17292904/d03a4440-7ff2-48ad-81a6-de30c11b1bf1)

![Bildschirmfoto 2024-03-22 um 09 28 44](https://github.com/GetPublii/Publii/assets/17292904/5c3ad7dc-b88b-4cdd-b330-f0dc105ba34f)

**Comment by bobmitro** [comment]
“Zoom effect” refers to the zoom feature that is activated when you hover over photos. I think you are talking about the "object-fit" for images. The "object-fit" property is for controlling how images fit within their container, not zooming. It keeps the image proportions correct.

**Reply by 3muchfun** [reply]
I am not talking about the image but the heading. In my example the word "Kos", which moves up a bit when I hover over the image or the word.

**Reply by bobmitro** [reply]
How does the zoom effect in photos have anything to do with it? I don't understand it. It's the hover effect: the heading goes up when you hover on the cards.

**Reply by 3muchfun** [reply, marked as answer]
OK, here we are: What I meant, is this behavior of the text/link underneath the featured image:

```
.c-card__link:hover .c-card__text {
  opacity: 0.6;
  -webkit-transform: translateY(-0.5333333333rem);
          transform: translateY(-0.5333333333rem);
  will-change: transform;
}
```

Anyway, I set this to zero, what fits better, in my opinion.
